### DISCUSSION TITLE
How do I set up an Ubuntu computer as a harvester?

### DISCUSSION THREAD

**Discussion by 459577107** [Chia-Network/chia-blockchain#3084] [Q&A]
Hello, I am new to Chia and Linux and I cannot run Chia-h command under Linux.
What is the reason, please?

**Comment by SackofFluor** [comment]
Do you have any additional information?
Does return any errors?
Are you running it as root?

**Reply by 459577107** [reply]
Yes. I use ubuntu GUI and install the fron the link in red circle

![1619693126](https://user-images.githubusercontent.com/6745742/116539244-27179480-a91b-11eb-8e76-45126ad77069.jpg)

**Comment by Cast39** [comment]
Did you follow these steps?
https://github.com/Chia-Network/chia-blockchain/wiki/INSTALL#ubuntudebian

did you type in the **. ./activate** command in the chia-blockchain directory? you have to do this every time you restart or open a new console.
aka there should be a (venv) in front of your bash:
![Screenshot from 2021-04-29 12-40-02](https://user-images.githubusercontent.com/29007078/116538641-14d03300-a8e8-11eb-9053-1bae4660204b.png)

**Reply by 459577107** [reply]
Hello friend. Thank you for your replying. 
I install in the red circle.
![1619693126](https://user-images.githubusercontent.com/6745742/116539355-49111700-a91b-11eb-9ffe-4e2c140900a8.jpg)
Am I right? I cant find ../activate in under the chia-blockchain.

**Reply by Cast39** [reply]
Sorry but i have to ask. Have you noticed the space between the two dots in . ./activate and the space between chia and -h -> (chia -h)?

**Reply by 459577107** [reply]
> Sorry but i have to ask. Have you noticed the space between the two dots in . ./activate and the space between chia and -h -> (chia -h)?

Thank for your patience.
![image](https://user-images.githubusercontent.com/6745742/116540202-6abece00-a91c-11eb-88eb-b05d06f371a3.png)
The red circle means there is no such a file. 
I just consider that if I need install it by command  rather than GUI.

**Reply by Cast39** [reply]
okay i think you don't need to do this because you installed it as a package.

could you send the result of **whereis chia** and **chia -h**? Maybe this shows some weird things.

alternatively you could remove the current installation (when you already have plots, save them and the 32 or so words for your key) and install it with the other method. This is also not that hard, just copy the commands one by one.

**Reply by 459577107** [reply]
> okay i think you don't need to do this because you installed it as a package.
> 
> could you send the result of **whereis chia** and **chia -h**? Maybe this shows some weird things.
> 
> alternatively you could remove the current installation (when you already have plots, save them and the 32 or so words for your key) and install it with the other method. This is also not that hard, just copy the commands one by one.

Yes I will try to  reinstall by command. Because I cant find ./activate.
Thanks to you and good luck for me.

**Reply by 459577107** [reply]
I have install but there is something happen when I run chia init
<img width="606" alt="1619702034(1)" src="https://user-images.githubusercontent.com/6745742/116556472-0c9be600-a930-11eb-99ee-7ca2c6ed9305.png">
Did I do something stupid again? hahaha~

**Reply by Cast39** [reply]
no this is perfectly normal xD you have to add a key now xD. Did you also install the gui? (You can set up a key in there and also farm, this should be the easiest) The instructions for starting the gui are also in the install instructions

**Reply by 459577107** [reply]
> no this is perfectly normal xD you have to add a key now xD. Did you also install the gui? (You can set up a key in there and also farm, this should be the easiest) The instructions for starting the gui are also in the install instructions

Friend. I want to make this machine as a harvester.
But when I enter f key and p key from my main machine. It has been start to plot.....
So... My question is that why I need ca if it could use f key and p key to plot....
